crates/harness/src/monolith.rs

```diff
@@ -416,6 +421,7 @@ pub struct MockRequest {
 pub struct MonolithBuilder {
     response_mocks: HashMap<String, (MockRespParts, Bytes)>,
     behavior: Option<Box<dyn Behavior + Send + 'static>>,
+    region: String,
```

Comment:
I'd like the default for region to be `unknown` instead of an empty string. I would recommend just doing this in `new()`.